app/src/main/java/com/sopt/now/compose/Friend.kt

```diff
@@ -0,0 +1,8 @@
+package com.sopt.now.compose
+
+data class Friend(
+    val profileImageRes: Int,
```

Comment:
```suggestion
    @DrawableRes val profileImageRes: Int,
```

리소스 아이디의 경우에는 앞에 이런식으로 어노테이션 붙이면 좋을 것 같습니다.

추가로 해당 아이템이 로컬 이미지가 아닌 외부 저장소에서 가져와야 하는 이미지라면 패러미터가 Int만 받으면 안될텐데 어떻게 대응할 수 있을까요?

Reply:
헉 외부에서 가져와야 하는 경우는 생각해보지 못했습니다...
찾아보겠습니다!